packages/messaging-api-messenger/src/Messenger.js

```diff
@@ -0,0 +1,41 @@
+import type {
+  UserID,
+  Recipient,
+  SendOption,
+  Message,
+  SendMessageSucessResponse,
+} from './MessengerTypes';
+
+export default class Messenger {
+  static createRequest = (
+    body: Object
+  ): Promise<SendMessageSucessResponse> => ({
```

Comment:
Does this function return `Promise<SendMessageSucessResponse>` or `BatchItem`?

Reply:
fixed